src/input/inputModule.py

```diff
@@ -11,8 +13,34 @@ def all_symbol_names(self) -> list[str]:
         """Return every symbol that is imported when importing * from the library"""
         return []
 
-    def visit_call(self, func_name: str, args: list, kwargs: dict) -> IRNode:
-        pass
+    def visit_method(self, func_name: str, args: list, kwargs: dict) -> Optional[IRNode]:
```

Comment:
you probably mean visit_function, since you call a function imported from pandas, while visit method would mean that you call a real method on an object (I know you can call pandas.read_sql) but still I will less confused if this method is called visit_function

Reply:
yeah, i never really got the point of functions and methods not being synonyms. But yes, let's stick to the usual naming